services/php-apache/Dockerfile

```
@@ -3,9 +3,10 @@ FROM tugboatqa/php:${PHP_VERSION}-apache
 ENV COMPOSER_DISCARD_CHANGES=true
 ENV COMPOSER_NO_INTERACTION=true
 ENV COMPOSER_ALLOW_SUPERUSER=true
-RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
-    && curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add - \
-    && echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list \
+RUN git clone --branch v8.0.1 https://github.com/tj/n.git \
+    && cd n && make install && cd .. \
+    && n lts \
```

Let’s change this to install Node v10 by default, since that is the version that this image already provided by default. This will guarantee we don’t break anything for any projects using the `php-apache` image while we sort out what the lift will be to switch those projects to use `php-apache-node` if they _actually need_ Node.
```suggestion
    && n 10 \
```

---

But I thought what you were really suggesting was, instead of this change, remove the Node installation from this image completely?

---

I am suggesting that, but we do not know the scope of the impact of doing so. How many projects would we need to switch over to use `php-apache-node`? So I’m leaving this suggestion in the meantime.

---

Hence “short-term compromise”

---

Got it!